react/react1/week2/src/pages/NotePage.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from "react";
+import Note from "../components/Notes/Note";
+import NoteForm from "../components/Notes/NotesForm";
+
+function NotePage() {
+  const [noteValue, setNoteValue] = useState({
+    description: "",
+    deadline: "",
+  });
+  const [tasks, setTasks] = useState([]);
+
+  useEffect(() => {
+    const fetchTasks = async () => {
+      const response = await fetch(
+        "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw"
+      );
+      const data = await response.json();
+      setTasks(data);
+    };
+    fetchTasks();
+  }, []);
```

Comment:
Great! You can even do it without fetchTasks function, just adding async on the useEffect callback